src/bootstrap/compile.rs

```diff
@@ -106,14 +106,17 @@ pub fn std_link(build: &Build,
     t!(fs::create_dir_all(&libdir));
     add_to_sysroot(&out_dir, &libdir);
 
-    if target.contains("musl") && !target.contains("mips") {
```

Comment:
Oh I thought we were replacing the `contains("mips")` annotations with a different one? Otherwise this may end up breaking the mips-musl build :(